frontend/src/pages/LoginPage.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <wof-card>
+    <div class="login-page">
+      <h1 class="login-page__title">Login</h1>
+      <div class="horizontal-line"></div>
+      <form class="login-page__form">
+        <div class="login-page__input">
+          <wof-input name="User name" :error="username.errorMsg" @change="setUsername" :maxLength="20"></wof-input>
+        </div>
+        <div class="login-page__input">
+          <wof-input name="Password" :type="'password'" :error="password.errorMsg" @change="setPassword"></wof-input>
+        </div>
+      </form>
+      <div class="horizontal-line"></div>
+      <div class="login-page__bottom">
+        <router-link to="/register" class="login-page__link">
+          Don't have an account? Sign Up
+        </router-link>
+        <wof-button :size="1.5" class="login-page__submit-button" @click="logIn">
+          Login
+        </wof-button>
+      </div>
+    </div>
+  </wof-card>
+</template>
+
+<script>
+import WofInput from "../components/WofInput.vue";
+
+export default {
+  name: "LoginPage",
+  components: { WofInput },
+  data() {
+    return {
+      username: {
+        value: '',
+        errorMsg: ''
+      },
+      password: {
+        value: '',
+        errorMsg: ''
+      },
+    };
+  },
+  methods: {
+    setUsername(newValue) {
+      this.username.value = newValue;
+    },
+    setPassword(newValue) {
+      this.password.value = newValue;
+    },
+    formValidation() {
+     if(this.username.errorMsg == '' && this.password.errorMsg == '') {
+        if(this.username.value.length > 0 && this.password.value.length > 0) {
+          return true;
+        }
+      }
+      return false;
+    },
+    logIn() {
+      if(this.formValidation()) {
+        //This will be replaced with call to http
+        console.log("Username:"+this.username.value);
+        console.log("Password:"+this.password.value);
+      }
+    }
+  }
+};
+</script>
+
+<style lang="less" scoped>
+@import "../components/common.less";
+
+.login-page {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 80vw;
```

Comment:
Mhm, that's a bit much for the login isn't it? Just looking at the login pages of other site I notice a tendency to have a smaller login component, e.g. here's a gmail login page:
![image](https://user-images.githubusercontent.com/24480246/143684667-c60b679a-c6c5-435f-8382-c65ba2efa74e.png)

And here's ours:
![image](https://user-images.githubusercontent.com/24480246/143684680-a95b2840-7b7f-4cdb-9386-28cf5e622b2b.png)

Reply:
We might also think about renaming login to sign in (a bit more grammatically correct).

Reply:
Yeah, i know this is big. I will think about it and

Reply:
I changed it and also changed Register Page (i will maybe think about making common style for these forms).
It looks a little better now, but we don't want (or rather say - we don't have enough time) to make this site fully responsive